src/components/sponsors.component.ts

```diff
@@ -0,0 +1,44 @@
+import { Component } from '@angular/core';
+
+import CatolicaUniversityLogo from '@assets/images/sponsors/universidad-catolica-logo.webp';
+
+@Component({
+  selector: 'app-sponsors',
+  standalone: true,
+  template: `
+    <article class="py-12 bg-[#1A1A1A]">
```

Comment:
we can use variables for this. Here is a reference: https://tailwindcss.com/docs/theme